core/pkg/ingress/controller/controller.go

```diff
@@ -944,9 +956,6 @@ func (ic *GenericController) createServers(data []interface{},
 						servers[host].SSLCertificate = cert.PemFileName
 						servers[host].SSLPemChecksum = cert.PemSHA
 					}
-				} else {
-					servers[host].SSLCertificate = defaultPemFileName
-					servers[host].SSLPemChecksum = defaultPemSHA
 				}
```

Comment:
This was removed due to a request from @bprashanth to not assign the default certificate on misconfigured ingresses.

Instead, I'm going to create a new ConfigMap to make this behaviour configurable (a heroku like), and it will be an admin choice to enable or not this.